content/en/apps/guides/forms/wealth-quintiles.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Tracking Wealth Quintiles"
+linkTitle: "Quintiles"
+weight: 
+description: >
+  How to track wealth quintiles on the profile of each family member in the household
+relatedContent: >
+
+---
+
+
+A few things are needed to get the data copied onto the family members. 
+
+1. A report submitted with either `doc.fields.NationalQuintile` or `doc.fields.UrbanQuintile` or both. 
+1. An offline user that has the permission of `can_write_wealth_quintiles`. 
+1. Once the form is submitted the contact will then get the fields `"wealth_quintile_national"` and ` "wealth_quintile_urban"` if they are provided in the form.
+
+
+### Sample Report
+
+
+``` json
+{
+  "form": "family_survey",
+  "type": "data_record",
+  "content_type": "xml",
+  "fields": {
+    "place_id": "23c7ab6e-6ea5-4f9f-9ffa-f00a1e30278d",
+    "place_name": "Medic Area",
```

Comment:
The `place_name` isn't required and it may be confusing here.
```suggestion
```

Reply:
👍